### PR TITLE
Add PY2 support

### PR DESCRIPTION
**Pull request by mvsusp**
*Description of changes:*
- add Python 2 support

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://hbybmzuba2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=53550a3e-9e2e-486c-ba6d-ed1b974cc812%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://hbybmzuba2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=1aa3d445-432b-4081-8f8f-f64984aade1f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://hbybmzuba2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=1d57eb3f-5f53-48dd-81c7-3d6f3e561183%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://hbybmzuba2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=c924ef5b-2073-45f2-a33f-6384426e61e3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: buildspec.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 0.2
 env:
   variables:
     FRAMEWORK_VERSION: '1.1.0'
-    CPU_PY_VERSION: '3'
```
Comment (nskool) [inline buildspec.yml]: Why is this removed?

File: buildspec.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: 0.2
...
+    CPU_PY_VERSION: '2'
     CPU_INSTANCE_TYPE: 'ml.c4.xlarge'
     GPU_PY_VERSION: '3'
```
Comment (nskool) [inline buildspec.yml]: GPU_PY_VERSION: '2'?
